code/game/turfs/turf.dm

```diff
@@ -21,10 +21,10 @@
 
 	var/explosion_level = 0	//for preventing explosion dodging
 	var/explosion_id = 0
-	
+
 	var/list/decals
 
-/turf/New()
+/turf/Initialize()
 	..()
 
 	levelupdate()
```

Comment:
The queue_smooth inside this proc accounts for 1/4 of ssobject's initialization.

since SSicon_smooth initializes later on and already handles smoothing zlevels 1 and 2 on initialize(), it might be worth the snowflake to have queue_smooth do nothing if the turf would be smooth'ed when SSicon_smooth initializes (and SSicon_smooth hasn't initialized yet.)

Overall I love this pr! Mainly because now we can see and easily profile the cost our turfs were having on world init.

When are you going to make all objs use initialize? =P